conftest.py

```diff
@@ -26,7 +26,7 @@
 hypothesis.settings.register_profile(
     'fuzzing', deadline=None, print_blob=True, max_examples=1000
 )
-default = 'fuzzing' if (os.environ.get('TRAVIS_EVENT_TYPE') == 'cron' and os.environ.get('TRAVIS_CPU_ARCH') != 'arm64') else 'ci'  # noqa: E501
+default = 'fuzzing' if (os.environ.get('IS_CRON') == 'true' and os.environ.get('ARCH_ON_CI') not in ('aarch64', 'ppc64le')) else 'ci'  # noqa: E501
```

Comment:
Could we print this in the pytest header or just directly so it's easy to see in the CI logs ?

Reply:
Great idea. I added something, let's hope it works.